src/Orleans.Transactions.Azure/Storage/AzureTransactionLogOptions.cs

```diff
@@ -7,4 +10,26 @@ public class AzureTransactionLogOptions
 
         public string TableName { get; set; } = "TransactionLog";
     }
+
+    public class AzureTransactionLogOptionsValidator : IConfigurationValidator
+    {
+        private readonly AzureTransactionLogOptions options;
+
+        public AzureTransactionLogOptionsValidator(IOptions<AzureTransactionLogOptions> configurationOptions)
+        {
+            this.options = configurationOptions.Value;
+        }
+
+        public void ValidateConfiguration()
+        {
+            if (string.IsNullOrWhiteSpace(this.options.ConnectionString))
+            {
+                throw new OrleansConfigurationException($"The azure transaction log was configured with an invalid connection string");
```

Comment:
nit: Azure should be capitalized. Should we point out how to configure the ConnectionString/TableName values in these errors or is it too obvious?

Reply:
Made error message a bit more explicit.